config/services.yaml

```
@@ -0,0 +1,43 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices.html#use-parameters-for-application-configuration
+parameters:
+  mlp.packages.dir: '%kernel.project_dir%/packages'
+  mlp.translations.dir: '%kernel.project_dir%/translations'
+  mlp.transifex.api.token: '%env(TRANSIFEX_API_TOKEN)%'
+  mlp.transifex.organisation: '%env(TRANSIFEX_ORGANISATION)%'
+  mlp.transifex.project: '%env(TRANSIFEX_PROJECT)%'
+  mlp.aws.key: '%env(AWS_KEY)%'
+  mlp.aws.secret: '%env(AWS_SECRET)%'
+  mlp.aws.region: '%env(AWS_REGION)%'
+  mlp.aws.bucket: '%env(AWS_BUCKET)%'
+
+services: # default configuration for services in *this* file
+  _defaults:
+    autowire: true      # Automatically injects dependencies in your services.
+    autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+
+  # makes classes in src/ available to be used as services
+  # this creates a service per class whose id is the fully-qualified class name
+  MauticLanguagePacker\:
+    resource: '../src/'
+    exclude:
+      - '../src/DependencyInjection/'
+      - '../src/Entity/'
+      - '../src/Kernel.php'
+
+  # add more service definitions when explicit configuration is needed
+  # please note that last definitions always *replace* previous ones
+
+  mautic.transifex:
+    class: Mautic\Transifex\Transifex
+    factory: [ 'MauticLanguagePacker\Factory\TransifexFactory', 'create' ]
+    arguments:
+      $apiToken: '%mlp.transifex.api.token%'
+      $organisation: '%mlp.transifex.organisation%'
+      $project: '%mlp.transifex.project%'
+
+  Mautic\Transifex\TransifexInterface:
+    alias: 'mautic.transifex'
```

```suggestion
  Psr\Http\Client\ClientInterface:
    class: GuzzleHttp\Client

  Psr\Http\Message\RequestFactoryInterface:
    class: Http\Factory\Guzzle\RequestFactory

  Psr\Http\Message\StreamFactoryInterface:
    class: Http\Factory\Guzzle\StreamFactory

  Psr\Http\Message\UriFactoryInterface:
    class: Http\Factory\Guzzle\UriFactory

  Mautic\Transifex\Config:
    calls:
      - [ 'setApiToken', [ '%env(TRANSIFEX_API_TOKEN)%' ] ]
      - [ 'setOrganization', [ '%env(TRANSIFEX_ORGANISATION)%' ] ]
      - [ 'setProject', [ '%env(TRANSIFEX_PROJECT)%' ] ]

  Mautic\Transifex\TransifexInterface:
    class: Mautic\Transifex\Transifex
```
We might save some lines and make it more straightforward as you define these services in services_test.yaml.